_data/utilities.yml

```diff
@@ -285,6 +285,13 @@ websites:
       img: sprint.png
       tfa: No
 
+    - name: Swisscom
+      url: https://www.swisscom.ch/
+      twitter: Swisscom_de
+      facebook: Swisscom
+      img: swisscom.png
+      tfa: No
+
```

Comment:
Since the Twitter speaks German, could you please add a `lang: de` tag?

Reply:
Yeah, sure. I added it to my fork.